### DISCUSSION TITLE
Is there any plan to transition webpack to rspack?

### DISCUSSION THREAD

**Discussion by zhaohuanyuu** [gatsbyjs/gatsby#37731] [Ideas / Feature Requests] [answered]
rspack doc: https://www.rspack.dev/

Now that rspack is officially released, is it possible to transform it on the existing basis in the future?

**Comment by LekoArts** [comment, marked as answer]
Please see https://github.com/gatsbyjs/gatsby/pull/37732

> We're still exploring Rspack inside Gatsby and we aren't making any definitive decisions yet. However, we're happy with the results so far and want to continue exploring Rspack for Gatsby.
